### ISSUE TITLE
Broken links on Open Water Analytics Intro Page

### ISSUE THREAD

**Issue by LRossman** [OpenWaterAnalytics/EPANET#764]
While this issue is not directly related to this code repository I wasn't sure where else to raise it. The OWA intro web page at [http://wateranalytics.org](http://wateranalytics.org) has some broken links on it:
- The link to the Forums ([http://community.wateranalytics.org](http://community.wateranalytics.org)) no longer works. This loss of the forum is unfortunate since it was a useful resource for practitioners.
- The link to the next-net ([https://github.com/OpenWaterAnalytics/next-net](https://github.com/OpenWaterAnalytics/next-net)) project is broken.

Is there any way to get the OWA Forum restored (or has GitHub eliminated this feature)?

**Comment by Mariosmsk** [comment]
> The link to the Forums (http://community.wateranalytics.org/) no longer works. This loss of the forum is unfortunate since it was a useful resource for practitioners.

I fully agree with that. Also, I already sent an email to @eladsal and @samhatchett.
